### PR TITLE
Articles Style Adjustments

### LINKED ISSUE #73: Main Page Style Refactor
Fix minor style issues on the main page.

### PR DESCRIPTION
**Pull request by JayDampitan**
## Changes
1. Added titles on every Articles.
2. Fix Layout on Movie Article to make it center. 
3. Format Articles with Prettier. 

Closes #73

### CODE REVIEW

File: the-toad-tribune/src/layout/Animals.tsx [hunk 1]
```diff
@@ -77,15 +77,12 @@ const Animals: React.FC<INewsProps> = ({
...
-
 const AnimalsStyles = styled.div<DarkModeProps>`
   grid-area: 6/6/10/8;
   overflow: hidden;
   display: flex;
   align-items: center;
   margin-bottom: 0.5em;
   cursor: pointer;
-  color: ${props => props.darkMode ? "#e3dac9" : "#1a1a1a"};
+  color: ${(props) => (props.darkMode ? "#e3dac9" : "#1a1a1a")};
```
Comment (Lap343) [inline the-toad-tribune/src/layout/Animals.tsx:87]: Dark mode! Nice.

Comment (gonzalespaulb) [inline the-toad-tribune/src/layout/Animals.tsx:87]: Snazzy

File: the-toad-tribune/src/layout/Sports.tsx [hunk 2]
```diff
@@ -26,40 +26,39 @@ const Sports: React.FC<INewsProps> = ({
...
+      <Buttons
+        onClick={() => {
+          pageNumber === 0
+            ? startEnd(articleResponse.articles.length - 1)
+            : onPrevButton();
+        }}
+        className="add-class"
+      >
+        <img src={PrevIcon} alt="" />
+      </Buttons>
```
Comment (JorgeLVilla) [inline the-toad-tribune/src/layout/Sports.tsx:38]: Nice!

**Review by gonzalespaulb — APPROVED** [review]

**Review by JorgeLVilla — APPROVED** [review]
Nice job Jay!

**Review by Lap343 — APPROVED** [review]
Good job man. Code looks good!

**Pull request merged**